profile/src/main/resources/HowToAddSupportForNewSchemaVersion.md

```diff
@@ -0,0 +1,49 @@
+## How to add support for a new schema version
```

Comment:
Is this the best place for this file? Perhaps it should be somewhere in docs/developer